sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -530,8 +530,7 @@
     },
     "/jobs/create": {
       "post": {
-        "summary": "Create a new import/export job on an IoT hub.",
-        "description": "Create a new import/export job on an IoT hub. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "Create a new import or export job on your IoT Hub. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities for more information.",
```

Comment:
Sorry for the back and forth @barustum but I think "your" is probably not the best and most consistent article to use. I would like to know your opinion too. Sometime we use "the", sometimes "an", sometimes "your". My thought is "the" is the safest. Might be a good thing to discuss at standup too.

Reply:
I also am not sure what the best word to use, or the recommended guidelines for SDK documentation is. Let's discuss in scrum tomorrow.

Reply:
I reworded to "Create a new import or export job on Iot Hub". Generic enough until we find a solution :). Changed other instances as well.